exporting/opentsdb/metadata.yaml

```diff
@@ -0,0 +1,139 @@
+id: 'export-opentsdb'
+meta:
+  name: 'OpenTSDB'
+  link: 'https://github.com/OpenTSDB/opentsdb'
+  categories: 
+	- export
+  icon_filename: 'opentsdb.png'
+keywords: 
+	- exporter
+	- OpenTSDB
+	- scalable time series
+overview:
+  exporter_description: |
+	Use the OpenTSDB connector for the exporting engine to archive your Netdata metrics to OpenTSDB databases for long-term storage, further analysis, or correlation with data from other sources.
+  exporter_limitations: ''
+setup:
+  prerequisites:
+    list:
+      - title: ''
+        description: |
+	   - OpenTSDB and Netdata, installed, configured and operational
+  configuration:
+    file: 
+      name: 'exporting.conf'
+    options:
+      description: 'The following options can be defined for this exporter.'
+      folding:
+        title: 'Config options'
+        enabled: true
+      list:
+        - name: 'enabled'
+          default_value: 'no'
+          description: 'Enables or disables an exporting connector instance (yes|no).'
+          required: true
+        - name: 'destination'
+          default_value: 'no'
+          description: 'Accepts a space separated list of hostnames, IPs (IPv4 and IPv6) and ports to connect to. Netdata will use the first available to send the metrics..'
+          required: true
+          detailed_description: |
+	     The format of each item in this list, is: [PROTOCOL:]IP[:PORT].
+
+	     PROTOCOL can be udp or tcp. tcp is the default and only supported by the current exporting engine.
+
+	     IP can be XX.XX.XX.XX (IPv4), or [XX:XX...XX:XX] (IPv6). For IPv6 you can to enclose the IP in [] to separate it from the port.
+	
+	     PORT can be a number of a service name. If omitted, the default port for the exporting connector will be used (graphite = 2003, opentsdb = 4242).
```

Comment:
Why are you citing graphite port n opentsdb yaml file? :thinking: I think we can remove it.